src/test/java/seedu/address/model/task/deadline/DurationTest.java

```diff
@@ -19,9 +19,10 @@ public void isValidTitle() {
 
         // invalid duration
         assertFalse(Duration.isValidDuration(-10)); // negative number
```

Comment:
Test a few more values in each equivalence partition and especially near boundary values. Please include values such as 0, -1, some valid values and values near MAX_INT